operate/client/src/App/DecisionInstance/index.tsx

```diff
@@ -20,29 +17,17 @@ import {Header} from './Header';
 import {VariablesPanel} from './VariablesPanel';
 import {Forbidden} from 'modules/components/Forbidden';
 import {DrdPanel} from './DrdPanel';
-import {drdStore} from 'modules/stores/drd';
```

Comment:
@christoph-fricke Sorry if there was some miscommunication, but it's ok to keep Mobx stores for basic frontend states.

What we want to avoid is using for fetching data and having too complex stores which depend on each other.

Please reach out to me if you have any questions so we can clarify it. I think we can simply revert the changes to the `drdStore` for now

Reply:
@vsgoulart huh. The guidelines and "idea bulk" in the Operate frontend guide explains the goal to migrate away from MobX and replace everything with Tanstack query and React state. 🤔 https://github.com/camunda/operate/wiki/Operate-Front-End-%E2%80%90-Architecture#state-management

Reply:
Let's leave it then. This needs to be corrected. I'll sinc with Paddy when he's back